compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolEnv.java

```diff
@@ -75,6 +75,8 @@
 
     public int relativeEnvCount;
 
+    public boolean isModuleInit;
```

Comment:
Why are we adding this to the SymbolEnv? IMO this doesn't belong here. 

Looking at the usage, can't we just use a boolean in the DataflowAnalyzer which we set and unset in `analyzeModuleInitFunc`? Like we generally do in CodeAnalyzer? https://github.com/ballerina-platform/ballerina-lang/blob/6fc33a04d14a5d9d6fd49bd2507e5097fad2dfe5/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java#L252